bin/common/write_k8s_ip_to_environment.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -o errexit
+while ! ifconfig eth1 | grep -q addr; do
+  sleep 3
+done
+K8S_VM_IP=$(ifconfig eth1 | grep -oE 'addr:[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}' | sed 's/addr://g')
+echo K8S_VM_IP=$K8S_VM_IP >> /etc/environment
```

Comment:
Hmm, actually, I don't see this being used anywhere but cert generation.  So, that script?
But we might want to run the cert generation as a `ExecStartPre` in that case.  Or rather as a thing both apiserver and kublet and whatnot depends on. Which would be sad, but ah well.